sageresearch-app-sdk/schemas/org.sagebionetworks.research.sageresearch.dao.room.mock.MockResearchDatabase/1.json

```diff
@@ -0,0 +1,226 @@
+{
```

Comment:
Is this a new schema? Not sure where this came from

Reply:
I just tried to delete that file, it was added by mistake.